utils/values_test.py

```diff
@@ -60,7 +60,7 @@ def test_pval_identical(self):
 
   def test_pval_significant(self):
     values = Values([1, 1, 1])
-    self.assertAlmostEqual(0.967, values.pval([10, 10, 10]), places=3)
+    self.assertAlmostEqual(0.900, values.pval([10, 10, 10]), places=3)
```

Comment:
:eyes: ?

Reply:
The new scipy version produces a pval of 0.9, given the inputs.

Reply:
Updated the test to a sample size of 5.